website/homepage/src/guides/template.html

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    {% with title=pretty_name, url="https://gradio.app/" + name, image="/assets/img/meta-image.png", description="A Step-by-Step Gradio Tutorial" %}
+    {% with title=pretty_name, url="https://gradio.app/" + name, image="https://www.gradio.app/assets/img/meta-image.png", description="A Step-by-Step Gradio Tutorial" %}
```

Comment:
I think we also need to modify the `meta.html` template otherwise the twitter path is going to be messed up?

```
  | <meta property="og:image" content="https://www.gradio.app/assets/img/meta-image.png" />
  | <meta name="twitter:card" content="summary_large_image">
  | <meta name="twitter:creator" content="@Gradio">
  | <meta name="twitter:title" content="Using Blocks Like Functions">
  | <meta name="twitter:description" content="A Step-by-Step Gradio Tutorial.">
  | <meta name="twitter:image" content="https://gradio.app/https://www.gradio.app/assets/img/meta-image.png">
```

I'm actually not sure what the original problem is or how to test. @aliabd Do you know? 

Is it the double-slash that gets created in the twitter meta tag in the template?

In any event, maybe just changing the template to be
```
<meta name="twitter:image" content="{{ image }}">
```

Is the right thing to do.

Reply:
My bad, yeah you're right. 

I think Twitter actually looks at 

`<meta property="og:image" content="https://www.gradio.app/assets/img/meta-image.png" />`

not 

`  | <meta name="twitter:image" content="https://gradio.app/https://www.gradio.app/assets/img/meta-image.png">`

At least according to this: https://developer.twitter.com/en/docs/twitter-for-websites/cards/guides/getting-started

Reply:
Will fix after lunch